### PR TITLE
Budget App

### PR DESCRIPTION
**Pull request by iamchristianani**
#### Live Demo: [Link](https://bill-splitter-v1fd.onrender.com/)

> In this Pull Request, the following changes were made

- [x] Added the splash screen.
- [x] Added the Signup and Log-in pages.
- [x] Added Group (Categories) page.
- [x] Added Expenses (Transaction) page.
- [x] Added the "Add a new group" form page.
- [x] Added the "Add a new expense" form page.
- [x] Added unit test using RSpec.
- [x] Ensured the database table matches the [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/images/erd_diagram.png) diagram.
- [x] Ensured there was no linter error.
- [x] Deployed the app using [render](https://bill-splitter-v1fd.onrender.com/).
- [x] Updated my readme file.

### CODE REVIEW

**Comment by addod19** [comment]
### Hi @iamchristianani, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is invalid because the README file is missing the `future features` section

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app/views/groups/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<section>
+  <h2 class="text-center">Groups</h2>
+  <% unless @groups.empty? %>
+    <%= render 'list' %>
+  <% else %>
+    <p>No groups available</p>
+  <% end %>
+  <%= link_to 'Add a new group', new_group_path, class: 'btn bg-sec w-100 text-light' %>
```
Comment (Lucas-Masaba) [inline app/views/groups/index.html.erb]: - [ ] Please make sure this button looks similar like that provided in the original design. It should be at the bottom of the page with a white thin border surrounding the test.
### Your Implementation
image
### Required
image

As mentioned in the project requirements, you should follow all the [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:

* Colors
* Typography: font face, size and weight.
* Layout: composition and space between elements

File: app/views/expenses/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<section>
+  <h2 class="text-center">Expenses</h2>
+  <div class="text-secondary text-center">Group - <%= @group.name %></div>
+  <p>Total price: $<%= @group.total_price %></p>
+  <% if @expenses.blank? %>
+    <p>No Expenses found.</p>
+  <% else %>
+    <%= render 'list' %>
+  <% end %>
+  <%= link_to 'Add a new Expense', new_expense_path, class: 'btn bg-sec w-100 text-light' %>
```
Comment (Lucas-Masaba) [inline app/views/expenses/index.html.erb]: - [ ] Please make sure this button looks similar to that provided in the original design. It should be at the bottom of the page with a thin white border surrounding the test like I mentioned in the previous comment

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @iamchristianani  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Linter checks are passing :heavy_check_mark: 
- Tests are passing :heavy_check_mark: 
- The UI is generally good-looking:star: 
- Descriptive README :scroll: 
- Descriptive pull request :writing_hand: 
- Meaningful commit messages :surfer: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
Hi @iamchristianani,

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**